dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -440,5 +448,117 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
 
     }
 
+    /**
+     * Given sceneario: Multi-language content referenced from a page. Create a Site-Search run including all the languages of the content and
+     * check the two versions made it into the resulting index.
+     */
+
+    @Test
+    public void test_MultilangContent_IndexingAllLanguages()
+            throws DotPublishingException, JobExecutionException, DotDataException, IOException, DotSecurityException, WebAssetException {
+
+        boolean defaultContentToDefaultLangOriginalValue =
+                Config.getBooleanProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", false);
+
+        try {
+
+            Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", true);
+
+            final Host site = new SiteDataGen().nextPersisted();
+            Language lang1 = APILocator.getLanguageAPI().getDefaultLanguage();
+            Language lang2 = new LanguageDataGen().nextPersisted();
+            folder = new FolderDataGen().site(site).nextPersisted();
+
+            Contentlet contentletLang1 = TestDataUtils
+                    .getEmployeeContent(true, lang1.getId(), null, site);
+
+            contentletLang1 = contentletAPI.find(contentletLang1.getInode(), systemUser, false);
+            contentletLang1.setStringProperty("firstName", "catherine");
+            contentletLang1 = contentletAPI.checkin(contentletLang1, systemUser, false);
+
+            ContentletDataGen.publish(contentletLang1);
+
+            Contentlet contentletLang2 = contentletAPI
+                    .find(contentletLang1.getInode(), systemUser, false);
+            contentletLang2.setInode("");
+            contentletLang2.setLanguageId(lang2.getId());
+            contentletLang2.setStringProperty("firstName", "catalina");
+            contentletLang2 = contentletAPI.checkin(contentletLang2, systemUser, false);
+
+            ContentletDataGen.publish(contentletLang2);
+
+            final Container container = new ContainerDataGen().withContentType(contentletLang1
+                    .getContentType(), "$!{firstName}").nextPersisted();
+
+            ContainerDataGen.publish(container);
+
+            final String uuid = UUIDGenerator.generateUuid();
+
+            final Template template = new TemplateDataGen()
+                    .withContainer(container.getIdentifier(), uuid)
+                    .nextPersisted();
+
+            TemplateDataGen.publish(template);
+
+            HTMLPageAsset page = new HTMLPageDataGen(folder, template).languageId(lang1.getId())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'page' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5260866)